pulp_2to3_migration/app/pre_migration.py

```diff
@@ -621,14 +635,23 @@ def delete_old_resources(plan):
     old_dist_query &= Q(pulp2_repository__in=repos_with_old_distributions_qs) | Q(not_in_plan=True)
 
     with transaction.atomic():
-        pulp2distributors_with_old_distributions_qs = Pulp2Distributor.objects.filter(
-            old_dist_query).only('pulp3_distribution')
-
+        pulp2distributors_with_old_distributions_qs = Pulp2Distributor.objects.filter(old_dist_query)
+        pb_to_delete = set()
```

Comment:
Maybe it's just me, but the `pb` part is very confusing, the first association is progess bar.  I even had to look at the code to see where it's used.
 It's not critical but maybe `pub_to_delete` or the plural form might reflect better what it is, `pubs_to_delete`.

Reply:
sure, i can update that if it is confusing